core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2darrays/Print2DArraysUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.print2darrays;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.stream.SystemOut;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Remove unused import. Please format the code using the Baeldung formatter to avoid these issues:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij